easybuild/easyconfigs/s/SAGA/SAGA-9.2.0-foss-2022a.eb

```
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'SAGA'
+version = '9.2.0'
+
+homepage = 'https://saga-gis.sourceforge.io'
+description = """
+SAGA is the abbreviation for System for Automated Geoscientific Analyses.
+SAGA is a Geographic Information System (GIS) software.
+SAGA has been designed for an easy and effective implementation of spatial algorithms.
+SAGA offers a comprehensive, growing set of geoscientific methods.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['http://sourceforge.net/projects/saga-gis/files/SAGA%20-%209/SAGA%20-%209.2.0/']
```

it's cumbersome but 
```suggestion
source_urls = [
    'http://sourceforge.net/projects/saga-gis/files/SAGA%%20-%%20%s/SAGA%%20-%%20%%(version)s' % (version.split('.')[0])
]
```

allows fetching with `--try-software-version` to work (tested with 9.1.3 and 8.5.1)